.clang-format

```diff
@@ -0,0 +1,151 @@
+## This config file is only relevant for clang-format version 8.0.0
```

Comment:
Please do not copy the `.clang-format` repository into the repository. We have a GitHub Action that will fetch the latest version from the ITK repository, which avoids duplication and getting out of sync.